index.html

```diff
@@ -1102,21 +1085,67 @@
           modifiers for particular payment method identifiers. For example, it
           allows you to adjust the total amount based on payment method.
         </dd>
+      </dl>
+    </section>
+    <section data-dfn-for="PaymentDetailsInit" data-link-for=
+    "PaymentDetailsInit">
+      <h2>
+        <dfn>PaymentDetailsInit</dfn> dictionary
+      </h2>
+      <pre class="idl">
+        dictionary PaymentDetailsInit : PaymentDetailsBase {
+          required PaymentItem total;
+        };
+      </pre>
+      <p>
+        The <a>PaymentDetailsInit</a> dictionary is used in the construction of
+        the payment request.
+      </p>
+      <p>
+        In addition to the members inherited from the <a>PaymentDetailsBase</a>
+        dictionary, the following members are part of the
+        <a>PaymentDetailsInit</a> dictionary:
+      </p>
+      <dl>
+        <dt>
+          <dfn>total</dfn>
+        </dt>
+        <dd>
+          See <a>PaymentDetailsBase.total</a>.
```

Comment:
DictionaryName.memberName notation is not great and should not be exposed publicly. I'd do `<a>PaymentDetailsBase</a>'s <a data-lt="PaymentDetailsBase.total">total</a>`.